docs/usage.md

```diff
@@ -62,6 +62,48 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Creating a branch
+
+Use `git-branch` command to create a new branch on base repo and any enabled modules.
+To create a new branch, you can use the **checkout** option along with the revision
+you want to work with. If you want to create a new branch with a certain name,
+specify the branch name to create with the `-b` or `--branch` option.
+
+Checkout to the revision you want to work with:
+```bash
+evg-module-manager git-branch --operation checkout --revision "revision_to_checkout"
+```
+
+To create a branch named `my-branch`, use the following:
+```bash
+evg-module-manager git-branch --operation checkout --branch my-branch --revision HEAD
+```
+
+
+## Updating existing git branches
+
+Use `git-branch` command to update your existing branches on base repo and any enabled modules.
+By default, a git checkout action will be performed to the user provided revision
+in the base repository.
+
+For revisions on modules, the revision to be checked out would be the
+revision for the module in the evergreen manifest associated with the latest commit in the history that
+was run in evergreen.
+
+To update the branch with the latest comment on all enabled modules. You can use the **rebase** or
+**merge** operations, if any merge conflicts occur, they will be reported and
+the repository will be left in the unmerged state for manually resolution.
+
+To merge my active branch on the most recent comment:
```

Comment:
These examples don't really make sense. Merging or rebasing on HEAD should be a no-op. Since HEAD refers to the current latest commit.